README.rst

```diff
@@ -609,6 +609,39 @@ Cache can be disabled via ``--using-cache`` option or config file:
         ->setUsingCache(false)
     ;
 
+Cache file can be specified via ``--cache-file`` option or config file:
+
+.. code-block:: php
+
+    <?php
+
+    return Symfony\CS\Config\Config::create()
+        ->setCacheFile(__DIR__.'/.php_cs.cache')
+    ;
+
+Speed up builds on Travis
```

Comment:
Both approaches have strong advantages and in the end the repo owner decides how he want to deal with it.
And StyleCI is great service for sure.
Just PHP CS Fixer's readme isn't great place for StyleCI how-to.